.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 extension
-extension.zip
+DS_Store
```

Comment:
```suggestion
.DS_Store
```

Reply:
.DS_Store should not be ignored in project gitignores, only your global gitignore (see [Configuring ignored files for all repositories on your computer](https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer))